modules/serverless/pages/features/transport-encryption/transport-encryption-setup.adoc

```diff
@@ -0,0 +1,226 @@
+= Setup transport encryption in Eventing
+:compat-mode!:
+// Metadata:
+:description: Setup {serverlessproductname} transport encryption for Eventing
+
+This page describes the transport encryption feature which allows transporting data and events over secured and encrypted HTTPS connections using TLS.
+
+[IMPORTANT]
+====
+{serverlessproductname} transport encryption for Eventing is a Developer Preview feature only.
+Developer Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete.
+Red Hat does not recommend using them in production.
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Developer Preview features, see https://access.redhat.com/support/offerings/devpreview/.
+====
+
+== Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+
+* Install the OpenShift CLI (`oc`).
+
+* Install the {serverlessoperatorname}.
+
+* Install the {certmanageroperatorname}.
```

Comment:
```suggestion
* OpenShift CLI (`oc`) is installed.

* {serverlessoperatorname} is installed.

* {certmanageroperatorname} is installed.
```

nit. Maybe easier to read as it is a "prerequisite" :shrug:

Reply:
I copied it from here https://openshift-knative.github.io/docs/docs/latest/serverless/service-mesh/common-service-mesh-setup.html

![image](https://github.com/openshift-knative/docs/assets/33736985/2cdf45fb-6cc1-4f4c-be1c-b9b32d6dfa8a)